packages/patternlab/styleguide/source/_meta/_00-head.twig

```diff
@@ -8,6 +8,8 @@
     <link rel="stylesheet" href="../../assets/css/index-generated.css?{{ cacheBuster }}" media="all" />
 
     <script src="../../assets/js/vendor/modernizr.js"></script>
+    
+    <script src="https://cdn.userway.org/widget.js" data-account="zgT5fBBncI"></script>
```

Comment:
@clairesunstudio Does this only get loaded for the Patternlab preview pages in Mayflower? This script load wouldn't be carried over to openmass, right?

Reply:
Not seeing this script in the latest update for mayflower-artifacts, so I think we are good to go.